util.py

```diff
@@ -100,7 +104,10 @@ def get_user_points() -> Dict[str, int]:
       commit_username, = match.groups()
 
       if commit_username in points:
-        points[commit_username] += 1
+        points[commit_username]['merge'] += 1
+
+  for user in points:
```

Comment:
Maybe `points` should be its own object, rather than a `Dict`, so that `total` can be a property, since it's a derived value.